src/gui/vector/qgsattributesformmodel.cpp

```diff
@@ -0,0 +1,1162 @@
+}
+
+void QgsAttributesFormTreeData::DnDTreeNodeData::setVisibilityExpression( const QgsOptionalExpression &visibilityExpression )
+{
+  mVisibilityExpression = visibilityExpression;
+}
+
+QgsOptionalExpression QgsAttributesFormTreeData::DnDTreeNodeData::collapsedExpression() const
+{
+  return mCollapsedExpression;
+}
+
+void QgsAttributesFormTreeData::DnDTreeNodeData::setCollapsedExpression( const QgsOptionalExpression &collapsedExpression )
+{
+  mCollapsedExpression = collapsedExpression;
+}
+
+QgsAttributesFormTreeData::RelationEditorConfiguration QgsAttributesFormTreeData::DnDTreeNodeData::relationEditorConfiguration() const
+{
+  return mRelationEditorConfiguration;
+}
+
+void QgsAttributesFormTreeData::DnDTreeNodeData::setRelationEditorConfiguration( const QgsAttributesFormTreeData::RelationEditorConfiguration &relationEditorConfiguration )
+{
+  mRelationEditorConfiguration = relationEditorConfiguration;
+}
+
+QgsAttributesFormTreeData::QmlElementEditorConfiguration QgsAttributesFormTreeData::DnDTreeNodeData::qmlElementEditorConfiguration() const
+{
+  return mQmlElementEditorConfiguration;
+}
+
+void QgsAttributesFormTreeData::DnDTreeNodeData::setQmlElementEditorConfiguration( const QmlElementEditorConfiguration &qmlElementEditorConfiguration )
+{
+  mQmlElementEditorConfiguration = qmlElementEditorConfiguration;
+}
+
+
+QgsAttributesFormTreeData::HtmlElementEditorConfiguration QgsAttributesFormTreeData::DnDTreeNodeData::htmlElementEditorConfiguration() const
+{
+  return mHtmlElementEditorConfiguration;
+}
+
+void QgsAttributesFormTreeData::DnDTreeNodeData::setHtmlElementEditorConfiguration( const HtmlElementEditorConfiguration &htmlElementEditorConfiguration )
+{
+  mHtmlElementEditorConfiguration = htmlElementEditorConfiguration;
+}
+
+QgsAttributesFormTreeData::SpacerElementEditorConfiguration QgsAttributesFormTreeData::DnDTreeNodeData::spacerElementEditorConfiguration() const
+{
+  return mSpacerElementEditorConfiguration;
+}
+
+void QgsAttributesFormTreeData::DnDTreeNodeData::setSpacerElementEditorConfiguration( SpacerElementEditorConfiguration spacerElementEditorConfiguration )
+{
+  mSpacerElementEditorConfiguration = spacerElementEditorConfiguration;
+}
+
+QColor QgsAttributesFormTreeData::DnDTreeNodeData::backgroundColor() const
+{
+  return mBackgroundColor;
+}
+
+void QgsAttributesFormTreeData::DnDTreeNodeData::setBackgroundColor( const QColor &backgroundColor )
+{
+  mBackgroundColor = backgroundColor;
+}
+
+QgsAttributesFormTreeData::TextElementEditorConfiguration QgsAttributesFormTreeData::DnDTreeNodeData::textElementEditorConfiguration() const
+{
+  return mTextElementEditorConfiguration;
+}
+
+void QgsAttributesFormTreeData::DnDTreeNodeData::setTextElementEditorConfiguration( const QgsAttributesFormTreeData::TextElementEditorConfiguration &textElementEditorConfiguration )
+{
+  mTextElementEditorConfiguration = textElementEditorConfiguration;
+}
+
+//
+// QgsAttributesFormTreeNode implementation
+//
```

Comment:
Mix of
/*
 *
 */

and
//
//
//

We'd need to fix that, but I think we just want them out :)